CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.15)
 PROJECT(whole_body_state_msgs)
 
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
```

Comment:
I think we can remove the if loop. Is there something I am missing here?

Reply:
It appears we can do without this (at least on my machine). I've commented it out for now.

Reply:
You can still keep "add_compile_options". It is indeed a good idea!